.github/workflows/tests.yml

```diff
@@ -393,40 +482,38 @@ jobs:
 
     runs-on: ubuntu-latest
     steps:
-      - name: Download test results
-        uses: actions/download-artifact@v4
+      - name: Download Artifacts
+        uses: actions/download-artifact@v3
 
-      - name: Upload combined test results
-        # provides one downloadable archive of all .coverage/test-report.xml files
-        # of all matrix runs for further analysis.
+      - name: Upload Combined Test Results
+        # provides one downloadable archive of all matrix run test results for further analysis
         uses: actions/upload-artifact@v4
         with:
           name: test-results-${{ github.sha }}-all
-          path: test-results-${{ github.sha }}-*
-          retention-days: 90  # default: 90
+          path: test-results-*
+          retention-days: 7  # for durations.yml workflow
 
       - name: Test Summary
         uses: test-summary/action@v2
         with:
-          paths: ./test-results-${{ github.sha }}-**/test-report*.xml
+          paths: test-results-*/test-report.xml
 
   # required check
   analyze:
-    name: Analyze results
     needs: [windows, linux, linux-qemu, linux-benchmarks, macos, aggregate]
     if: '!cancelled()'
 
     runs-on: ubuntu-latest
     steps:
-      - name: Decide whether the needed jobs succeeded or failed
-        uses: re-actors/alls-green@05ac9388f0aebcb5727afa17fcccfecd6f8ec5fe
+      - name: Determine Success
+        uses: re-actors/alls-green@v1.2.2
```

Comment:
Dependabot usually recommends pinning the hash to avoid tag replacement attacks. Dependabot can recognize the hashes as version approach and update them as necessary too, so I'd recommend switching back to hashes in 3rd party (i.e. no `actions/*`) actions.

Reply:
From my experience I haven't seen dependabot updating hashes when a new version was available, perhaps theres some config option we need to enable?

Reply:
See this one PR https://github.com/conda-forge/docker-images/pull/256. [Config here](https://github.com/conda-forge/docker-images/blob/main/.github/dependabot.yml).

Reply:
ah good good, I've opened https://github.com/conda/conda/issues/13657 to address this